scripts/fix_vdom_constructor_usage.py

```diff
@@ -1,370 +0,0 @@
-from __future__ import annotations
```

Comment:
Might want to keep this file, depending on whether we want to auto migrate `key=...`

Reply:
I think it'd probably be simpler to just have a normal deprecation warning for that. I can always go back in the history to find this as well.

Reply:
I'm actually going to handle moving key to be an attribute. There's a lot of places in the docs I don't want to fix so I may actually write a script to auto-update those things.